NEWS.md

```diff
@@ -16,6 +16,12 @@ Compiler/Runtime improvements
 Command-line option changes
 ---------------------------
 
+* The entry point for Julia has been standardized to `Main.main(ARGS)`. When julia is invoked to run a script or expression
+(i.e. using `julia script.jl` or `julia -e expr`), julia will subsequently run the `Main.main` function automatically if
+such a function has been defined. This is identended to unify script and compilation workflows, where code loading may happen
+in the compiler and execution of `Main.main` may happen in the resulting executable. For interactive use, there is no semantic
+difference between defining a main function and executing the code directly at the end of the script. ([50974])
```

Comment:
```suggestion
* The entry point for Julia has been standardized to `Main.main(ARGS)`. When Julia is invoked to run a script or expression
(i.e. using `julia script.jl` or `julia -e expr`), Julia will subsequently run the `Main.main` function automatically if
such a function has been defined. This is intended to unify script and compilation workflows, where code loading may happen
in the compiler and execution of `Main.main` may happen in the resulting executable. For interactive use, there is no semantic
difference between defining a `main` function and executing the code directly at the end of the script. ([50974])
```

Reply:
This is talking about the behavior of the binary, not the semantics of the language, so lower case is the appropriate style.